src/css/profile/mobile/common/radio.less

```diff
@@ -7,40 +7,38 @@ input[type="radio"].ui-radio {
 	-webkit-appearance: none;
 	margin: 0 18 * @px_base;
 
-	&::before,
-	&::after {
+	&::before {
 		content: "";
 		position: absolute;
-		width: 100%;
+		left: 0;
+		top: 0;
+		width: 32 * @px_base;
+		height: 32 * @px_base;
+		background-color: var(--ripple-color);
+		border-radius: 100%;
+		opacity: 0;
 	}
 
-	&::before {
+	&::after {
+		content: "";
+		position: absolute;
 		height: 100%;
-		opacity: 0.8;
-		background-color: var(--control-inactive-color);
+		width: 100%;
+		bottom: 0;
+		box-sizing: content-box;
+		background-color: var(--control-active-color);
 		mask-image: url("images/3_Controllers/Radio/sem_btn_radio_to_on_mtrl_000.svg"); 
 		mask-size: 100%, 0;
```

Comment:
This is out of scope your change but should be `mask-size: 100%` remove ', 0' . This property refers to next mask url.

Reply:
done